packages/documentation-framework/scripts/md/parseMD.js

```diff
@@ -281,9 +281,14 @@ function writeIndex() {
 }
 
 // Build unique names for components if they are a part of the "next" module.
```

Comment:
If we keep this function as you've expanded upon it, we should update this comment.